crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -83,12 +83,16 @@ impl Extension for LoggerExtension {
             .filter(|(_, operation)| operation.node.ty == OperationType::Query)
             .any(|(_, operation)| operation.node.selection_set.node.items.iter().any(|selection| matches!(&selection.node, Selection::Field(field) if field.node.name.node == "__schema")));
         if !is_schema && self.config.log_request_query {
-            info!(
-                query_id = ctx.data_unchecked::<Uuid>().to_string(),
-                session_id = ctx.data_unchecked::<SocketAddr>().to_string(),
-                "[Query] {}",
-                ctx.stringify_execute_doc(&document, variables)
-            );
+            let query_id: &Uuid = ctx.data_unchecked();
+            let session_id: &SocketAddr = ctx.data_unchecked();
+            if !is_health_check_request(query_id) {
```

Comment:
You already have a guard condition on L85 for this trace, so I would encourage you to re-use that forthis condition (by pulling the calls to `ctx.dat_unchecked()` out of the condition).